sbin/build.sh

```diff
@@ -1652,7 +1652,8 @@ addJ9Tag() {
   # This code makes sure that a version number is always present in the release file i.e. openj9-0.21.0
   local j9Location="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/openj9"
   # Pull the tag associated with the J9 commit being used
-  J9_TAG=$(git -C $j9Location describe --abbrev=0)
+  # ignore error if no tag
+  J9_TAG=$(git -C $j9Location describe --abbrev=0 2>/dev/null || true)
```

Comment:
Why this change?

Reply:
To avoid build failure if the openj9 has no tag. Useful for developer/experimental builds.

Reply:
>The command finds the most recent tag that is reachable from a commit. If the tag points to the commit, then only the tag is shown. Otherwise, it suffixes the tag name with the number of additional commits on top of the tagged object and the abbreviated object name of the most recent commit. The result is a "human-readable" object name which can also be used to identify the commit to other git commands.


Adding abbrev=0 will trim to only show the tag.
Have you seen a case where no tag is found? Shouldn't there always be a tag?
I am a bit worried about accidentally hiding a failure and what the implications are if J9_TAG is blank.

Reply:
```
09:48:18  checkoutRequiredCodeToBuild:
09:48:18    workspace = /home/jenkins/workspace/build-scripts/jobs/jdk/jdk-linux-ppc64le-openj9/workspace/./build//src
09:48:18    BUILD_VARIANT = openj9
09:48:18    TAG = 
09:48:18    BRANCH = openj9
...
10:19:56  ADDING J9 TAG <--- no tag, rc=128, build fails
[Pipeline] }
...
[Pipeline] // node
[Pipeline] }
[Pipeline] // stage
[Pipeline] echo
10:20:29  Execution error: hudson.AbortException: script returned exit code 128
```
see [jdk-linux-ppc64le-openj9](https://hyc-runtimes-jenkins.swg-devops.com/view/Adopt-Semeru/job/build-scripts/job/jobs/job/jdk/job/jdk-linux-ppc64le-openj9/97/)

The J9_TAG is used for setting variant_version json (in metadata). When J9_TAG='', variant_version={ "major": "0","minor": "0","security": "0","tags": "" }. 
Main repo is usually tagged but developers forks most likely not. For developer builds, this is a show stopper (a jdk and jre are produced but no artifacts would be available).  A warning could be added here.